frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2121,8 +2124,10 @@ doIsCandidateApplicableInitial(Context* context,
       // calling a field accessor
       auto ct = ci.actual(0).type().type()->getCompositeType();
       CHPL_ASSERT(ct);
-      CHPL_ASSERT(isNameOfField(context, ci.name(), ct));
-      return fieldAccessor(context, ct, ci.name());
+      CHPL_ASSERT(isNameOfField(context, ci.name(), ct) != nullptr);
```

Comment:
I think this assertion is redundant since we make the same check (just with the function call result put into `containingType`) immediately below

Reply:
good point, thanks.